libs/sdk/android/src/main/java/com/rudderstack/react/android/LifeCycleEvents.java

```diff
@@ -8,10 +14,62 @@
 public class LifeCycleEvents {
 
     interface LifeCycleEventsInterface {
-        public void run();
+        void run();
     }
 
-    static List<LifeCycleEventsInterface> lifeCycleEvents = new ArrayList<>();
+    private static final List<LifeCycleEventsInterface> lifeCycleEvents = new ArrayList<>();
+
+    static void trackAutomaticEvents() {
+        for (LifeCycleEventsInterface lifeCycleEvents : LifeCycleEvents.lifeCycleEvents) {
+            lifeCycleEvents.run();
+        }
+    }
+
+    static class ApplicationInstalledOrUpdatedRunnable implements LifeCycleEventsInterface {
```

Comment:
Rename LifeCycleEventsInterface to LifeCycleJob

Reply:
That interface belongs to Facebook react-native code, so cannot change the name!